pwa-devdocs/src/tutorials/pwa-studio-fundamentals/modify-site-footer/index.md

```diff
@@ -242,6 +242,37 @@ Open your project's `src/index.js` file and update the import for the App compon
 + import App from './components/App';
 ```
 
+If your file `src/index.js` doesn't have the import for **App, {AppContextProvider}**, you probably have the import of Adapter component. 
```

Comment:
```suggestion
If your `src/index.js` file doesn't define the `import` statement for **App, {AppContextProvider}**, it might instead define an `import` statement for the Adapter component. 
```

Reply:
~~What do you mean by "you probably have the import of Adapter component"?~~ 
Figured it out after reading through the rest of the topic. :)